forum/sharing.md

```diff
@@ -0,0 +1,39 @@
+# Sharing your Mods
+
+Once you've created a mod that you're proud of, the next step is to share it. Sharing will be a big help to other people. It will also help you use best practices and get help on your code. But it may also mean you have to respond to bugs in your code and help other people use your code.
+
+
+## Code headers
+
+Putting a comment at the top of your code helps people understand it, use it, and remember where it came from. Consider using this template:
+```
+/* ----------------------------------------------------------------
```

Comment:
Should we even version mods, or use the last compatible Vivaldi version? I think using the last compatible Vivaldi version combined with the last update date (should we use American or European style dates) could be a better solution.

Reply:
There’s no reason for custom versioning, but some people do. For them it’s a good idea to use something like

    Version: 2.1 for Vivaldi 3.9.2289.3

I prefer yyyy-mm-dd format for date, coz nobody can confuse that. (I dislike both of the formats you mentioned, coz they don’t go in the right direction: y>m>d>h>m>s, but dd/mm/yyyy hh:mm:ss or even mm/dd/yyyy ll:mm:ss p, but that’s personal again.)

Reply:
Please do pay attention to everything in the example comment:

- all lines except for the introducing one are indented by one space, this makes the 2nd asterisk in the introducing line go one character more to the right than the next lines.
- every line has an asterisk at char 2
- the padding between the asterisks and text is one space

Also, it might be better if the values are indented only as required by the longest key, but no further.

Reply:
Ok

Reply:
@ortiza5 @zalex108 what do you think about date formats?

Reply:
If dates must be used, as @tiosgz said, the `yyyy-mm-dd` or `yyyy/mm/dd` is probably best to avoid confusion. As someone from the US, there is too much ambiguity between the month and day with them in the beginning. You can't make a silent protest of our illogical date format by swapping month and day because everyone over here will confuse none obvious dates like `01/02/2021`.

I don't see a great need for dates in the header comment. Versions tested on should be enough.

This is what I proposed for header comments earlier:
```CSS
/**
 * <Title>
 * URL:         https://forum.vivaldi.net/topic/...
 * Description: <description>
 * Author(s):   <userID>
 * CopyRight:   No Copyright Reserved
 * Tested on:   Stable <version>  and  Snapshot <version>
 */
```

Reply:
Ok, that’s good. I do think dates in the header comment may be good, but I guess it’s fine because the forum and git both keep dates.